create-keystone-app/starter/schema.ts

```diff
@@ -1,94 +1,78 @@
 
-// We are using Typescript, and we want our types experience to be as strict as it can be.
-// By providing the Keystone generated `Lists` type to our lists object, we refine
-// our types to a stricter subset that is type-aware of other lists in our schema
-// that Typescript cannot easily infer.
-import { Lists } from '.keystone/types';
+// when using Typescript, you can refine your types to a stricter subset by importing
+// the generated types from '.keystone/types'
+import type { Lists } from '.keystone/types';
 
-// We have a users list, a blogs list, and tags for blog posts, so they can be filtered.
-// Each property on the exported object will become the name of a list (a.k.a. the `listKey`),
-// with the value being the definition of the list, including the fields.
 export const lists: Lists = {
-  // Here we define the user list.
   User: list({
-    // Here are the fields that `User` will have. We want an email and password so they can log in
-    // a name so we can refer to them, and a way to connect users to posts.
+    // WARNING
+    //   for this starter project, anyone can create, query, update and delete anything
+    //   if you want to prevent random people on the internet from accessing your data,
+    //   you can find out more at https://keystonejs.com/docs/guides/auth-and-access-control
+    access: allowAll,
+
+    // this is the fields for our User list
     fields: {
+      // by adding isRequired, we enforce that every User should have a name
+      //   if no name is provided, an error will be displayed
       name: text({ validation: { isRequired: true } }),
+
       email: text({
         validation: { isRequired: true },
+        // by adding isIndexed: 'unique', we're saying that no user can have the same
+        // email as another user - this may or may not be a good idea for your project
         isIndexed: 'unique',
-        isFilterable: true,
       }),
-      // The password field takes care of hiding details and hashing values
+
       password: password({ validation: { isRequired: true } }),
-      // Relationships allow us to reference other lists. In this case,
-      // we want a user to have many posts, and we are saying that the user
-      // should be referencable by the 'author' field of posts.
-      // Make sure you read the docs to understand how they work: https://keystonejs.com/docs/guides/relationships#understanding-relationships
+
+      // we can use this field to see what Posts this User has authored
+      //   more on that in the Post list below
       posts: relationship({ ref: 'Post.author', many: true }),
-    },
-    // Here we can configure the Admin UI. We want to show a user's name and posts in the Admin UI
-    ui: {
-      listView: {
-        initialColumns: ['name', 'posts'],
-      },
+
+      createdAt: timestamp({
+        // this sets the timestamp to Date.now() when the user is first created
+        defaultValue: { kind: 'now' }
```

Comment:
As an example of timestamp, this seemed more appropriate compared to `publishDate`